manifest/autoversion/autoversion.go

```diff
@@ -80,6 +86,44 @@ blocks:
 
 	// Update the manifest and write it out to disk.
 	content, err = hcl.MarshalAST(ast)
+
+	// prepare for PopulateDigestsForVersion
```

Comment:
This won't work correctly as there can be multiple auto-version blocks in a single manifest - that's what the above loop is doing. This will have to be taken into account.